attributes/default.rb

```diff
@@ -120,14 +120,14 @@
 
 default["certs"]["dir"]                            = node["install"]["dir"].empty? ? node["kagent"]["dir"] + "/certs-dir" : node["install"]["dir"] + "/certs-dir"
 
-default["tensorflow"]["version"]                   = "1.8.0"
+default["tensorflow"]["version"]                   = "1.10.0"
```

Comment:
This PR contains part of HOPSWORKS-638 - update to TensorFlow 1.10. According to this https://github.com/tensorflow/tensorflow/releases/tag/v1.10.0 we need to update also the NCCL version we are using to 2.2 (it's in the JIRA but it hasn't been done yet)